iceoryx_hoofs/vocabulary/include/iox/detail/string_internal.hpp

```diff
@@ -111,9 +107,14 @@ struct GetSize<char>
     }
 };
 
-/// @brief struct to get a pointer to the char array of the fixed string/string literal/std::string
+/// @brief struct to get a pointer to the char array of the fixed string/string literal
```

Comment:
```suggestion
/// @brief struct to get a pointer to the char array of the fixed string/string literal/custom string
```

Reply:
@elBoberido I moved the doxygen description to or custom struct to `std_string_support.hpp`.